cli/src/client/mod.rs

```diff
@@ -1,39 +1,30 @@
+use crate::errors::{ApolloError, ErrorDetails, Fallible};
 use graphql_client::{GraphQLQuery, Response};
 use http::Uri;
 use reqwest::blocking;
 use std::env;
-use std::error::Error;
 
 mod queries;
 
-static PROD_GQL_API_URL: &str = "https://engine-graphql.apollographql.com/api/graphql";
-
-fn api_uri() -> Uri {
-    env::var("APOLLO_API_URL")
```

Comment:
This line was lost -- there's no longer a way to set the URL.

This makes me wonder how the integration tests actually passed?

Reply:
We allow over-writing in the environment variable merge of config in `cli/src/config.rs:51`